spec.bs

```diff
@@ -478,9 +480,10 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
     [=auction config/per buyer experiment group ids=].
   1. If |perBuyerExperimentGroupIds| is not null and |perBuyerExperimentGroupIds|[|buyer|] exists:
     1. Set |buyerExperimentGroupId| to |perBuyerExperimentGroupIds|[|buyer|].
-  1. TODO: Let |perBuyerSignals| be null/empty
-  1. TODO: If |auctionConfig|'s [=auction config/per buyer signals=][|buyer|] exists:
-    1. TODO: Set |perBuyerSignals| be |auctionConfig|'s [=auction config/per buyer signals=][|buyer|].
+  1. If |auctionConfig|'s [=auction config/per buyer signals=][|buyer|] exists:

```

Comment:
Is `auction config/per buyer signals` guaranteed to be non-null here?

Reply:
Nope, added a null check here.